releasenotes/notes/fix-remove-iop-not-work.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: istioctl
+releaseNotes:
+  - |
+    **Fixed** `istioctl operator remove` not working properly.
```

Comment:
Could we be a little more specific here. Without having an issue or PR in the description I have no easy way to figure out what was fixed.

Reply:
Done.